JBTM-3091 Depend on a MP-LRA snapshot build

mmusgrov opened the pull request:
https://issues.jboss.org/browse/JBTM-3091

[microprofile-lra](https://github.com/eclipse/microprofile-lra) is still undergoing development and the APi is changing. Therefore the narayana implementation and quickstarts must track particular snapshot builds. This PR adds a dependency on snapshot builds.

There is also a small change from TimeUnit to ChronoUnit that was introduced by by an [API change](https://github.com/eclipse/microprofile-lra/issues/34)

tomjenkinson commented on rts/lra-examples/lra-coordinator/pom.xml:
```diff
@@ -87,5 +87,16 @@
             <artifactId>cdi</artifactId>
             <version>${version.wildfly-swarm}</version>
         </dependency>
+        <!-- generate REST API from swagger
```
Is this necessary to leave in?

mmusgrov replied:
As a quickstart I thought it would be useful for the reader to see what API our spec implementation uses to communicate with our coordinator. He may wish to experiment with non Java based clients and participants (nodejs for example).

tomjenkinson commented:
@mmusgrov looks like a hang: http://narayanaci1.eng.hst.ams2.redhat.com/job/btny-pulls-narayana-quickstart/123/

mmusgrov commented:
@tomjenkinson those were PR failures before I pushed 9a48d19 after which the PR jobs both passed.

tomjenkinson commented:
Ah OK  thanks

mmusgrov commented:
I have moved the swagger comment from the pom to the README. I also squashed the 3 commits made for this PR. I will wait for another CI run before merging.
